.travis.yml

```diff
@@ -33,7 +14,7 @@ script:
   - KERAS_BACKEND=tensorflow pytest -vvv --cov=minutes test
 
 after_success:
-  - python -m pip install python-coveralls
+  - flake8
```

Comment:
Any reason to lint after the tests run? Its nice  to catch linting errors early before a longish test run!

Reply:
True! I guess by putting the linting in `after_success`, the test suite is guaranteed to run, whereas when linting before the tests, Travis would stop at the first linting error it finds, and it would take one more (post-lint-fix) build before the failed test(s) were caught.

Our build only takes ~1 minute, so I don't think it's too bad if we run the test suite every time! 😛

Reply:
Only takes one minute for now 😬. Got ahead and merge! I have the last few fixes to #107 ready shortly.